pyproject.toml

```diff
@@ -221,6 +213,7 @@ max-complexity = 41
 "**/{examples}/*" = ["D103", "D205", "D301", "D400", "E402"]
 "**/{tests}/*" = ["D100", "D101", "D102", "D103", "E402", "PLR2004", "PLR0915"]
 "__init__.py" = ["D104", "E402"]
+"nilearn/datasets/**/*" = ["PTH201", "PTH101", "PTH102", "PTH107", "PTH110", "PTH111", "PTH113"]
```

Comment:
@michellewang 
I am ignoring some errors for the datasets subpackage here.
You may need want to check which ones you are solving when working on it.